api/zfile.api

```diff
@@ -20,6 +20,11 @@
         <argument name = "name" type = "string" />
     </constructor>
 
+    <constructor name="tmp">
```

Comment:
Could you please mark it as draft given it's new?

Reply:
Done

Reply:
great, thanks